prototype/mixins.py

```diff
@@ -31,6 +31,25 @@ def get_context_data(self, *args, **kwargs):
         )
 
 
+class GetCMSTagMixin(TemplateView):
```

Comment:
this does not need to inherit from `TemplateView` - it can be `class GetCMSTagMixin:` :)

Reply:
oh fair enough. i can do the same with `GetCMSPageByFullPathMixin`